### PR TITLE
Correcting RTL in Postinstall

### PR DESCRIPTION
**Pull request by infograf768**
### Summary of Changes
Postinstall component uses the admin `mod_feed` to display the **Release News from the Joomla Project.**
But it does not take into account a RTL language.
Alignment and direction of date are set by default to LTR
This PR corrects that by:
1. Settting the module `rssrtl` to RTL if lang is RTL for the envelopping `<div>` in postinstall default.php
2. Adding a specific style to the `<ul` to force the feed texts to be aligned left as these are in English in mod_feed.

### Testing Instructions
Install Persian language and switch admin to Persian.
Load `index.php?option=com_postinstall&eid=700`

### Before patch

The date in the left column is not displayed as RTL and is aligned left
Screen Shot 2019-07-17 at 11 30 41

### After patch
The date is now RTL and alignment is right
Screen Shot 2019-07-17 at 11 29 38

### Note
For 4.0 the modified code for mod_feed will be slightly different as we have no more 
`<?php echo $params->get('moduleclass_sfx'); ?>`
and the class for the feed texts is now `newsfeed list-group`

### CODE REVIEW

File: administrator/modules/mod_feed/tmpl/default.php [hunk 1]
```diff
@@ -83,7 +83,13 @@
 
 	<!-- Show items -->
 	<?php if (!empty($feed)) : ?>
-		<ul class="newsfeed<?php echo $params->get('moduleclass_sfx'); ?>">
+		<?php // postinstall override ?>
+		<?php if ($rssurl === 'https://www.joomla.org/announcements/release-news.feed?type=rss') : ?>
+			<?php $style = 'style= "text-align: left !important"'; ?>
```
Comment (Quy) [inline administrator/modules/mod_feed/tmpl/default.php]: Variable not necessary. Output directly in next line. Remove space after `=`.

Comment (infograf768) [reply administrator/modules/mod_feed/tmpl/default.php]: @Quy 
In fact, except for the space, this was to facilitate pushing this in 4.0 where we would have
```
		<?php // postinstall override ?>
		<?php if ($rssurl === 'https://www.joomla.org/announcements/release-news.feed?type=rss') : ?>
			<?php $style = 'style= "text-align: left !important"'; ?>
		<?php endif; ?>
		<ul class="newsfeed list-group" <?php echo $style; ?>>
```
and no else:

Shall I still change that?

Comment (Quy) [reply administrator/modules/mod_feed/tmpl/default.php]: OK. `$style` will always have to be defined.

Please fix the bullets. Also, may add padding to the right to not touch the box.

**Comment by ghost** [comment]
I have tested this item :red_circle: unsuccessfully on 0e91a5707e7181c2919568dd11e1fa194a626c24<br /><br />
#### without PR: Bullet is on Start of Title
ohne-pr
#### with PR: Bullet is on right End of Title (as Date).
mit-pr
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25604">issues.joomla.org/tracker/joomla-cms/25604</a>.</sub>

**Comment by infograf768** [comment]
I see other errors with the feed text which should be ltr. Looking at this now.
Also, the padding was in error before patch.

**Comment by infograf768** [comment]
@Quy @franz-wohlkoenig 
Should be OK now.

Screen Shot 2019-07-20 at 11 39 22

**Comment by ghost** [comment]
I have tested this item :red_circle: unsuccessfully on ce28993bf44d8f0009a1a3bd21eaeea919474b67<br /><br />All Paragraphs are ```<strong>```.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25604">issues.joomla.org/tracker/joomla-cms/25604</a>.</sub>

**Comment by infograf768** [comment]
@franz-wohlkoenig

This is not due to this patch but an error in the Persian ini file: a `<strong>` not closed properly:
 (`</ strong>`)
```
COM_PRIVACY_POSTINSTALL_BODY="<p>با معرفی GDPR برای شهروندان اتحادیه اروپا و مقررات مشابه در سایر نقاط جهان، ممکن است لازم باشد قبل از ذخیره هر <strong>اطلاعات شخصی</ strong> از کاربر رضایت بخواهید.</p><p>جوملا 3.9 قابلیت های جدیدی را برای کمک به شما در ایجاد سیاست های حفظ حریم خصوصی سایت و جمع آوری رضایت کاربر معرفی می کند. علاوه بر این، یک گردش کار برای کمک به مدیریت درخواستهای اطلاعات کاربر مانند درخواست حذف اطلاعات شخصی خود از سایت شما در دسترس است.</p><p>برای اطلاعات بیشتر در مورد این ویژگی جدید لینک <a href='https://docs.joomla.org/J3.x:Privacy' target='_new'>مستندات حریم خصوصی</a> را مطالعه کنید.</p>"
````
once corrected to 
```
COM_PRIVACY_POSTINSTALL_BODY="<p>با معرفی GDPR برای شهروندان اتحادیه اروپا و مقررات مشابه در سایر نقاط جهان، ممکن است لازم باشد قبل از ذخیره هر <strong>اطلاعات شخصی</strong> از کاربر رضایت بخواهید.</p><p>جوملا 3.9 قابلیت های جدیدی را برای کمک به شما در ایجاد سیاست های حفظ حریم خصوصی سایت و جمع آوری رضایت کاربر معرفی می کند. علاوه بر این، یک گردش کار برای کمک به مدیریت درخواستهای اطلاعات کاربر مانند درخواست حذف اطلاعات شخصی خود از سایت شما در دسترس است.</p><p>برای اطلاعات بیشتر در مورد این ویژگی جدید لینک <a href='https://docs.joomla.org/J3.x:Privacy' target='_new'>مستندات حریم خصوصی</a> را مطالعه کنید.</p>"

```
all is fine:

Screen Shot 2019-07-23 at 11 55 00

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 53467885e175f094eca1281f2b98207f64cd3d15<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25604">issues.joomla.org/tracker/joomla-cms/25604</a>.</sub>

**Comment by infograf768** [comment]
@Quy
Please confirm so that we get this in in next release.

File: administrator/components/com_postinstall/views/messages/tmpl/default.php [hunk 2]
```diff
@@ -72,7 +76,7 @@
 		<?php endif; ?>
 <?php if ($this->eid == 700) : ?>
 	</div>
-	<div class="span4">
+	<div class="span4" <?php if ($lang->isRtl()) : echo 'style="padding-right: 20px;"'; endif; ?>>
```
Comment (Quy) [inline administrator/components/com_postinstall/views/messages/tmpl/default.php]:
```suggestion
	<div class="span4"<?php if ($lang->isRtl()) : ?> style="padding-right: 20px;"<?php endif; ?>>
```

**Comment by infograf768** [comment]
Modified as requested. 
@franz-wohlkoenig No need to test again.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 0317860b265007ecf26b0ee6e8a19ccb56fcc7b0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25604">issues.joomla.org/tracker/joomla-cms/25604</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25604">issues.joomla.org/tracker/joomla-cms/25604</a>.</sub>

**Comment by infograf768** [comment]
@HLeithner 
Please merge.

**Comment by HLeithner** [comment]
thx

**Pull request merged**